src/main/java/com/tetrahedron/app/htmlApp/ViewScene.java

```diff
@@ -0,0 +1,43 @@
+package com.tetrahedron.app.htmlApp;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.Objects;
+
+public class ViewScene extends Application {
```

Comment:
Maybe lets rename it to TetrahedronApp
TO make it more class that it's a main class of our program